package.json

```diff
@@ -57,16 +57,26 @@
     "typescript": "^5.0.4"
   },
   "scripts": {
-    "build": "tsc",
-    "watch": "tsc -w",
-    "start": "doppler run -- yarn node --enable-source-maps dist/index.js",
+    "start": "docker compose -f ./docker/docker-compose.yml up -d",

```

Comment:
is it really necessary to run a container when developing something? Doesn't this take more time?

Reply:
Yes, it is necessary. It makes it possible to have the same development environment as in production.

Slower? Yes, but only during the build that must be done first and has a change of environment variable, that's all.

Reply:
don't you have to restart the whole process when you change files that are not part of the Client? like utils

Reply:
Yes, but it will restart directly because it would not need to rebuild